Add note for non productive environment. (#86)
4teamwork/ftw.linkchecker: README.rst

**************

It's important that this package isn't started by cronjob in non-production

> deployment. See non-production-info_ for more information.

Reviewer: So the point is that in a ZEO setup where the ZEO server might be down longterm for valid reasons - it could lock up/crash the entire machine.  Non zeo setups (e.g. development) would be fine as bin/instance isn't a ZEO Client.

I think I'd say.....

*Important:* You should be careful not to activate this script as a cronjob in environments where the ZEO server could be legitimately down for long periods (e.g. Staging / Test servers) as it could lead to locking up/crashing the entire machine.

What do you think?

Author: Sounds good: Adapted in 6badd5c

This package is copyright by `4teamwork <http://www.4teamwork.ch/>`_.

``ftw.linkchecker`` is licensed under GNU General Public License, version 2.

Additional Notes
================

.. _non-production-info:

Do not run in non-production
****************************

> In our setup, bin/instance is a so called ZEO client.

Reviewer: Just clarifying that this paragraph is talking about ZEO setups (which we don't use in development)

```suggestion

In development bin/instance is (usually) the Plone server.
In other setups, bin/instance is a so called ZEO client.
```

Author: Adapted in 6badd5c

A ZEO client will, instead of directly opening a Data.fs,
access the ZEO server over the network.
In our setups, this is wired up via ftw-buildouts.

Now, if the ZEO server cannot be reached (not running,
network issues, misconfiguration, ...), the ZEO client will
sleep for a bit, and try to reconnect.
By default, it does this in an infinite loop and it will
try to reconnect to the mothership until the end of time.

> For the regular instances (ZEP clients) running in supervisor,

Reviewer: ```suggestion
For the regular instances (ZEO clients) running in supervisor,
```

Author: Adapted in 6badd5c

this is the ideal behavior: If the ZEO server temporarily cannot
be reached, the clients will try to reconnect all by themselves.

> If the ZEO comes back up again, the system will fix itself without

Reviewer: ```suggestion
If the ZEO server comes back up again, the system will fix itself without
```

Author: Adapted in 6badd5c
